client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/checklist-item.tsx

```diff
@@ -5,31 +5,39 @@ import { isTaskDisabled } from './task-helper';
 import { Task } from './types';
 
 const ChecklistItem = ( { task }: { task: Task } ) => {
-	const { id, isCompleted, actionUrl, title, actionDispatch } = task;
+	const { id, isCompleted, keepActive, actionUrl, title, actionDispatch } = task;
 	const action = actionDispatch ? { onClick: actionDispatch } : { href: actionUrl };
 	const taskDisabled = isTaskDisabled( task );
+
+	let taskStyle = '';
+	if ( isCompleted ) {
+		taskStyle = 'is-completed';
+		if ( keepActive ) {
+			taskStyle += ' keep-active';
+		}
+	}
```

Comment:
I think the `classnames` import could simplify this.

Reply:
Thanks @Addison-Stavlo, updated